components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -54,6 +54,12 @@ import { CachingBlobServiceClientProvider } from '@gitpod/content-service/lib/su
 import { IDEOptions } from '@gitpod/gitpod-protocol/lib/ide-protocol';
 import { IDEConfigService } from '../ide-config';
 
+// shortcut
+export const traceWI = (ctx: TraceContext, wi: Omit<LogContext, "userId">) => TraceContext.setOWI(ctx, wi);    // userId is already taken care of in WebsocketConnectionManager
+export const traceAPIParams = (ctx: TraceContext, params: { [key: string]: any }) => TraceContext.addJsonRPCParameters(ctx, params);
```

Comment:
Thought about this as well. But IMO there is a semantic difference between a `workspaceId` parameter being traced and an being reported as `OWI` (owner-workspace-instance) - at least in my head. :stuck_out_tongue: 
 - OWI is about the user-workspace-instance at hand, as specific as possible, but _never_ mismatching non-owner `userId` with `workspaceId`
 - parameters is just parameters, it might related to any user/workspace

That being said, there is no documentation about this anywhere, yet. Mads started to write one, and I hope to have that discussion there.

I'd like to keep it as-is for now, clarify the tracing documentation in notion, and than maybe align later.

Reply:
> I'd like to keep it as-is for now, clarify the tracing documentation in notion, and than maybe align later.

Thanks for explaining your thought process! Works for me 👍 🚀